slay/code_gen.py

```diff
@@ -194,14 +194,7 @@ async def run(self, data: str, num_partitions: int) -> tuple[list, int]:
     """
     imports = ["from slay import stub"]
 
-    src_parts = [
-        f"""
-class {processor.cls_name}(stub.StubBase):
-
-    def __init__(self, url: str, api_key: str) -> None:
-        self._remote = stub.BasetenSession(url, api_key)
-"""
-    ]
+    src_parts = [f"class {processor.cls_name}(stub.StubBase):"]
```

Comment:
do you not need the constructor anymore?

Reply:
This is the class def, not the init, so it makes:
```
class GenerateData(stub.StubBase):

class SplitText(stub.StubBase):

class TextToNum(stub.StubBase):
```

But after this change we can leave out the `__init__`.